lib/rspec/core/configuration.rb

```diff
@@ -2196,15 +2219,27 @@ def run_suite_hooks(hook_description, hooks)
         end
       end
 
-      def get_files_to_run(paths)
```

Comment:
Why the extra method here?

Reply:
@JonRowe , I was thinking the method #get_files_to_run was getting too large, so I wanted to break it up.  Maybe that's premature, since nothing else calls #get_files.